docs/schema/identifiers.md

```diff
@@ -6,105 +6,107 @@ Consistent identifiers are essential to help join up open contracting data.
 
-* A release ID must be unique within the scope of the process of which it is a part;
-* Award and contract identifiers must be unique within the scope of the process of which they are a part;
-* An item, milestone or document ID must be unique within the array it is part of.
+````{admonition} Worked Example
+:class: hint
 
-Local identifiers must be used consistently. For example, if the `id` of an award is "22" in one release, then the `id` of the same award in another release must also be "22".
+Two publishers, the UK Atomic Energy Authority and Health Canada, use sequential numbers as internal identifiers for their contracting processes.
 
-## Open contracting process identifier (ocid)
+The UK Atomic Energy Authority initiates a contracting process to purchase productivity software and assigns it the internal identifier '0005'. Health Canada initiates a contracting process to purchase office furniture and also assigns it the internal identifier '0005'.
 
-<img src="../../_static/svg/green_compilation.svg" width="150" align="right"/>
+To create a globally unique `ocid`, each publisher prepends their internal identifier with their OCID prefix: 'ocds-fh349f' for the UK Atomic Energy Authority and 'ocds-twb234 for Health Canada.
 
-An open contracting process identifier (ocid) is a **globally unique identifier**. Every release has an `ocid`. OCDS defines an `ocid` as:
+The UK Atomic Energy Authority assigns the `ocid` 'ocds-fh349f-0005'.
 
-```{field-description} ../../build/current_lang/release-schema.json /properties/ocid
+```json
+{
+  "ocid": "ocds-fh349f-0005",
+  "publisher": {
+    "name": "UK Atomic Energy Authority"
+  },
+  "tender": {
+    "id": "0005",
+    "title": "Productivity software"
+  }
+}
 ```
 
-It can be used to join up information published at different times and in different places.
-
-Setting the `ocid` is usually a simple two step process:
-
-1. Identify the best **internal identifier** recorded against the processes being disclosed;
-2. Register an `ocid` prefix to prepend to this internal identifier.
+Health Canada assigns the `ocid` 'ocds-twb234-0005'.
 
-In some cases, you might need to consider changes to existing systems to ensure that different systems handling information about your contracting and planning processes have a common internal identifier to draw upon.
-
-```{admonition} Worked Example
-:class: hint
+```json
+{
+  "ocid": "ocds-twb234-0005",
+  "publisher": {
+    "name": "Health Canada"
+  },
+  "tender": {
+    "id": "0005",
+    "title": "Office furniture"
+  }
+}
+```
 
-In Mexico City, each time a tender or direct contract award process is initiated, the department responsible assigns an identifier.
+As a result, users and tools that work with data from both publishers are less likely to confuse the two contracting processes. If another publisher, such as an independent monitor, wants to publish more data about one of the contracting processes, they can reference it using its unique `ocid`.
```

Comment:
Removed content that overlaps with #364

```suggestion
As such, users and tools that work with both publishers' data will not confuse the two contracting processes.
```

The original content was:

> And, if an independent civil society contract monitoring group want to publish a report about implementation of Town A's road project, or Town B's text-book procurement, they have distinct identifiers they can use in their own data to refer to these.

But this is about references from a report or dataset, not from an OCDS dataset specifically.